Exercise_B/index.html

```diff
@@ -19,16 +24,73 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <div class="header">
+      <img src="../Exercise_B/assets/images/logo.png" alt="" />
+    </div>
+
     <!--Banner Jumbotron of the image-->
+    <div class="first-image">
+      <img
+        src="../Exercise_B/assets/images/flowershop.jpg"
+        class="banner"
+        alt=""
+      />
+      <div class="d-flex justify-content-center align-items-center">
+        <h1 class="display-4 title text-white">Beautiful Flowers</h1>
+        <button type="button" class="btn btn-learn text-white">
+          Learn more
+        </button>
+      </div>
+    </div>
 
     <!--Four containers with img / text-->
-    <!-- 4 containers -->
-
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
 
-    <!-- Delivery Div -->
+    <div class="footer d-flex justify-content-around">

```

Comment:
Hello Chris 
For this, i would suggest changing the grid
You can look at these for examples 
[https://getbootstrap.com/docs/5.0/layout/grid/](url)
Or try something like this 
`<div class="container-fluid">
	<div class="row">
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
	</div>
	<div class="row">
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
	</div>
</div>`

Reply:
`<div class="container-fluid">
<div class="row">
<div class="col"></div>
<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
	</div>
	<div class="row">
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
		<div class="col"></div>
	</div>
</div>`